index.html

```diff
@@ -51,36 +84,47 @@ <h1>Subidos</h1>
               <label>Hashtags: (separate by commas)
                 <textarea name="tags" placeholder="#nature, #sunset, #livinmybestlife"></textarea>
               </label>
-              <input type="submit" value="Upload">
+              <input type="submit" class="btn btn-success" value="Upload">
             </fieldset>
           </form>
-          <form id="change-pw">
+          <!-- <button id="pw-btn" type="button" class="btn btn-outline-success">Update Password</button> -->
+          <form id="change-pw" class="change-pw">
             <fieldset>
               <legend>Change Password</legend>
               <input name="passwords[old]" type="password" placeholder="Old Password">
               <input name="passwords[new]" type="password" placeholder="New Password">
-              <input type="submit" value="Change Password">
+              <input type="submit" class="btn btn-success" value="Change Password">
             </fieldset>
           </form>
           <form id="sign-out">
             <fieldset>
-              <legend>Sign out</legend>
-              <input type="submit" value="Sign out">
+              <input type="submit" class="btn btn-alert" value="Sign Out">
             </fieldset>
           </form>
         </nav>
 
-        <main>
-          <div class="modal fade" id="show-photo-modal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
-            <div class="modal-dialog modal-dialog-centered" role="document">
-              <div class="modal-content">
+        <div class="modal fade" id="show-photo-modal" tabindex="-1" role="dialog" aria-labelledby="showPhotoModalTitle" aria-hidden="true">
+          <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="modal-title" id="exampleModalLongTitle">Your Photo:</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div id="show-modal-body" class="modal-body"></div>
+              <div class="modal-footer">
```

Comment:
As far as I can tell, your show-photo modal has multiple headers and footers. The `show-display-owned.handlebars` already has its own header and footer. I'm not really sure what bootstrap does if you have a modal with multiple footers, but this could have something to do with your update not working.